fluidsynth/include/fluidsynth/event.h

```diff
@@ -56,6 +56,7 @@ enum fluid_seq_event_type {
   FLUID_SEQ_TIMER,		/**< Timer event (DOCME) */
   FLUID_SEQ_ANYCONTROLCHANGE,	/**< DOCME (used for remove_events only) */
   FLUID_SEQ_CHANNELPRESSURE,    /**< Channel aftertouch event @since 1.1.0 */
+  FLUID_SEQ_KEYPRESSURE,        /**< Polyphonic aftertouch event @since 1.1.7 */
```

Comment:
We are in a public header. Placing the key pressure next to channel pressure is absolutely comprehensible. Unfortunately this breaks the ABI. Applications compiled against older versions of fluidsynth will have different values for e.g. `FLUID_SEQ_SYSTEMRESET`.

Infact adding it anywhere else will always change the value of `FLUID_SEQ_LASTEVENT`, which might be used downstream.

Long story short: you can leave everything as it is, but this change makes fluidsynth require a `SOVERSION` bump, which will only happen upon a major release.